lib/generators/solidus_paypal_commerce_platform/install/install_generator.rb

```diff
@@ -92,6 +93,16 @@ def install_solidus_starter_frontend_support
         end
       end
 
+      def alert_no_classic_frontend_support
+        support_code_for(:classic_frontend) do
+          message = <<~TEXT
+            This extension do not support solidus_frontend. If you use it with solidus_frontend you need to build your own frontend code.
+            No frontend code has been copied to your application.
```

Comment:
Here we can suggest to look at version `~> 0.6` for support to solidus_frontend, that would be consistent with the solidus installer.